tensorflow_datasets/core/download/checksums.py

```diff
@@ -151,16 +151,14 @@ def _parse_url_infos(checksums_file: Iterable[str]) -> Dict[str, UrlInfo]:
     if not line or line.startswith('#'):
       continue
     # URL might have spaces inside, but size and checksum will not.
-    url, size, checksum = line.rsplit(' ', 2)
+    try:
+      url, size, checksum = line.rsplit('\t', 2)
+    except ValueError:
+      url, size, checksum = line.rsplit(' ', 2)
     url_infos[url] = UrlInfo(size=int(size), checksum=checksum)
   return url_infos
 
 
-def url_infos_from_path(checksums_path: str) -> Dict[str, UrlInfo]:
```

Comment:
Note that this function is called in other part of the code so this is breaking TFDS. I've removed this changed internally.

Reply:
@Conchylicultor Do i need to change the PR or is it alright?

Reply:
No need to update this PR